src/chocolatey/infrastructure.app/services/NugetService.cs

```diff
@@ -161,6 +161,22 @@ public virtual IEnumerable<PackageResult> List(ChocolateyConfiguration config)
                 }
             }
 
+            if ((config.ListCommand.ApprovedOnly || config.ListCommand.DownloadCacheAvailable || config.ListCommand.NotBroken) && config.RegularOutput)
+            {
+                // This warning has been added here, to provide context to the user, and a follow up issue
+
+                // has been added here: https://github.com/chocolatey/choco/issues/3139 to address the actual
+                // issue that causes this warning to be required.
+                this.Log().Warn(@"
+Starting vith Chocolatey CLI v2.0.0, changes have been made to the
+`choco search` command which means that filtering of packages using the

```

Comment:
```suggestion
choco search command which means that filtering of packages using the

```

Reply:
I would agree that backticks are used for this, but as it's going out in a warning message I'm not sure that makes sense? If it's consistent that we do this, then we should also add it to the options --approved-only, --download-cache, and --not-broken, mentioned below.

Reply:
I have had a look, and there is precedence for using backticks everywhere, so I have added them to the options.